.github/workflows/build.yml

```diff
@@ -30,4 +30,7 @@ jobs:
         run: ./gradlew clean build --refresh-dependencies -Pversion=$VERSION
         env:
           GITHUB_ACTOR: ${{ github.actor }}
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I wonder if these are necessary, and if so, what they are used for

Reply:
great question xD Let me remove them and see what happens :P